### PR TITLE
Changing instanceType in machineset on AWS takes effect only after scaling to 0 and back up

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1745132

### CODE REVIEW

**Comment by xltian** [comment]
@jhou1 can you pelase review from QE?

File: modules/machineset-modifying.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// 
+// * machine_management/modifying-machineset.adoc
+
+[id="machineset-modifying_{context}"]
+= Modifying a MachineSet
+
+When making changes to a MachineSet, you must remove all machines associated with the MachineSet 
+by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number.
```
Comment (jhou1) [inline modules/machineset-modifying.adoc]: We do not have to delete the machines, we just need to scale down the MachineSet. Deleting a machine will cause a new machine being created, if its replicas does not change.

Comment (ingvagabund) [reply modules/machineset-modifying.adoc]: Both are fine. New machine replacing deleted one will be created from updated template.

File: modules/machineset-modifying.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+----
+$ oc scale --replicas=2 machineset <machineset> -n openshift-machine-api
+----
+Or:
++
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
++
+Wait for the machines to start. The new machines use the updated MachineSet.
```
Comment (ingvagabund) [inline modules/machineset-modifying.adoc]: - `s/machines/Machines`
- The new Machines contain changes made to the Machineset's Machine template.

**Comment by mburke5678** [comment]
@jhou1 @ingvagabund What is the status of this PR? Thank you!

**Comment by jhou1** [comment]
Looks good to me.

File: modules/machineset-modifying.adoc (hunk 1 above)
Comment (ingvagabund) [reply modules/machineset-modifying.adoc]: > Does this show that the user needs to edit the YAML first, then remove the machines?

It's better phrased. lg. thanks.

File: modules/machineset-modifying.adoc (hunk 2 above)
Comment (ingvagabund) [reply modules/machineset-modifying.adoc]: looks good

File: modules/machineset-modifying.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Changes you make to a MachineSet do not affect existing machines. 
+
+If you need to scale a MachineSet without making other changes, you do not need
+to delete the machines.
+
+[NOTE]
+====
+By default, the {product-title} router pods are deployed on workers. Because the router 
+is required to access some cluster resources, including the web console, 
+do not scale the worker MachineSet to `0` unless you first relocate the router pods.
```
Comment (mburke5678) [inline modules/machineset-modifying.adoc]: @jhou1 @ingvagabund What is the best way to relocate the router pods? Such as using node selectors?

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: modules/machineset-modifying.adoc (hunk 3 above)
Comment (ingvagabund) [reply modules/machineset-modifying.adoc]: Out of my expertise, sorry :( Are the router pods deployed to every worker node or just some?

**Pull request merged**